nonspecific_current/neuron/leonhard.cpp

```diff
@@ -190,8 +324,15 @@ namespace neuron {
     }
 
 
-    inline double nrn_current_leonhard(size_t id, leonhard_Instance& inst, leonhard_NodeData& node_data, double v) {
+    inline double nrn_current_leonhard(_nrn_model_sorted_token const& _sorted_token, NrnThread* _nt, Memb_list* _ml_arg, int _type, size_t id, leonhard_Instance& inst, leonhard_NodeData& node_data, double v) {
+        _nrn_mechanism_cache_range _lmr{_sorted_token, *_nt, *_ml_arg, _type};
+        auto* const _ml = &_lmr;
+        auto* _thread = _ml_arg->_thread;
+        auto* _ppvar = _ml_arg->pdata[id];
         double current = 0.0;
+        func_leonhard(_ml, inst, id, _ppvar, _thread, _nt);
```

Comment:
This seems to be a sensible set of stuff to also pass into `nrn_current_leonhard`:
```
_ml, inst, id, _ppvar, _thread, _nt
```
(plus specific things).